torchaudio/models/__init__.py

```diff
@@ -1 +1,2 @@
 from .wav2letter import *
+from ._wavernn import *
```

Comment:
Because of this, we also need to prefix `ResBlock` and `MelResNet` with an underscore to make them both private too :)

Reply:
I have updated it.